teps/README.md

```diff
@@ -140,10 +140,17 @@ thus a feature) in the tektoncd projects.
    [the next API working group meeting](https://github.com/tektoncd/community/blob/main/working-groups.md#api)
    at the latest.
 
-Reviewers should use [`/approve`](../process.md#prow-commands) to indicate that they approve of the PR being merged. Once all assigned reviewers
-have approved the PR, the final [`/lgtm`](../process.md#prow-commands) can be added in
-[the next API working group meeting](https://github.com/tektoncd/community/blob/main/working-groups.md#api) or sooner
```

Comment:
the guideline here `or sooner by anyone with the permissions` facilitated merging TEPs offline ...

Reply:
I think it did in principle, but in practice, we encouraged people to check in with each other on slack if merging TEPs outside of API WG, even if nothing had changed. That's what I'm aiming to improve here

Reply:
Posting on a TEP slack channel is not a requirement, its just an option if the author or reviewers would like to chat in a group setting. The author and/or reviewers are more then welcome to chat via github comments or one-on-one or via any other means of communication.

I suggest keeping the original reference to the API WG as is:

```
Once all assigned reviewers
have approved the PR, the final [`/lgtm`](../process.md#prow-commands) can be added in
[the next API working group meeting](https://github.com/tektoncd/community/blob/main/working-groups.md#api) or sooner by anyone with permission if needed.
```

And now, we can elaborate on the **sooner**  or **offline** option. 

The reason I am requesting to keep API WG as the first option is creating a TEP is just another PR, and a PR just needs an approval and lgtm so that it can be merged. We do not have any automation to restrict and check if all the reviewers have approved the TEP. We have seen a couple of TEPs where just one reviewer had approved it and a collaborator (outside of list of reviewers) added `lgtm` which resulted in merging of those TEPs.